sos/plugins/origin.py

```diff
@@ -14,8 +14,8 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-from sos.plugins import Plugin, RedHatPlugin
 import os.path
+from sos.plugins import Plugin, RedHatPlugin
```

Comment:
Style note: typically we import the sos bits first, then python libs - I'll fix this up on merge.